rest-api-spec/src/main/resources/rest-api-spec/test/cat.shards/10_basic.yml

```diff
@@ -1,13 +1,13 @@
 "Help":
   - skip:
-      version: " - 2.11.99"
-      reason:  deleted docs added in 2.12.0
+      version: " - 2.99.99"
```

Comment:
Same, could you please add the test case for pre 2.12.0?

Reply:
There's already a test case in this file below for `"Help from 2.4.0 to 2.11.0":`. I don't think it makes sense to create a separate temporary `" - 2.99.99"` test case just for while we're doing the backport.